napari/_qt/widgets/qt_dims_slider.py

```diff
@@ -230,7 +230,8 @@ def _update_range(self):
 
     def _update_slider(self):
         """Update dimension slider."""
-        self.slider.setValue(self.dims.current_step[self.axis])
+        with self.dims.events.current_step.blocker():
```

Comment:
Curious if you added this defensively or because you ran into an issue?

Reply:
This is some leftover; it was needed because `current_step` was being deprecated but maintained, which caused an infinite loop here. Now I can safely remove this.